src/examples/patient-deduplication.test.ts

```diff
@@ -0,0 +1,170 @@
+import { Patient } from '@medplum/fhirtypes';
+import { MockClient } from '@medplum/mock';
+import { handler } from './patient-deduplication';
+
+describe('Link Patient', async () => {
+  test('Success', async () => {
+    const medplum = new MockClient();
+    //Create an original Patient
```

Comment:
Total nit, we should setup eslint and prettier on this repo.  There should be a space after `//`.

Reply:
Not really understanding why this doesn't work https://github.com/medplum/medplum-demo-bots/blob/main/package.json#L11